actions/schema-up/index.js

```diff
@@ -0,0 +1,111 @@
+const path = require('path');
+
+const {Toolkit} = require('actions-toolkit');
+const fetchSchema = require('./fetch-schema');
+
+Toolkit.run(async tools => {
+  tools.log.info('Fetching the latest GraphQL schema changes.');
+  await fetchSchema();
+
+  const {exitCode: hasSchemaChanges} = await tools.runInWorkspace(
+    'git', ['diff', '--quiet', '--', 'graphql/schema.graphql'],
+    {reject: false},
+  );
+  if (hasSchemaChanges === 0) {
+    tools.log.info('No schema changes to fetch.');
+    tools.exit.neutral('Nothing to do.');
+  }
+
+  tools.log.info('Committing schema changes.');
+  await tools.runInWorkspace('git', ['commit', '--all', '--message=":arrow_up: GraphQL schema"']);
+
+  tools.log.info('Re-running relay compiler.');
+  const {failed: relayFailed, all: relayOutput} = await tools.runInWorkspace(
+    path.resolve(__dirname, 'node_modules', '.bin', 'relay-compiler'),
+    ['--src', './lib', '--schema', 'graphql/schema.graphql'],
+    {reject: false},
+  );
+
+  const {exitCode: hasRelayChanges} = await tools.runInWorkspace(
+    'git', ['diff', '--quiet', '--', '**/__generated__/*.graphql.js'],
+    {reject: false},
+  );
+
+  if (hasRelayChanges === 0 && !relayFailed) {
+    tools.log.info('Generated relay files are unchanged.');
+    await tools.runInWorkspace('git', ['push']);
+    tools.exit.success('Schema is up to date on master.');
+  }
+
+  tools.log.info('Checking for unmerged schema update pull requests.');
+  const {data: openSchemaUpdatePRs} = await tools.github.graphql(`
+    query OpenPullRequests($owner: String!, $repo: String!) {
+      repository(owner: $owner, name: repo) {
+        pullRequests(first: 1, states: [OPEN], labels: ["schema update"]) {
```

Comment:
Readability suggestion: Encapsulate the label information in an object. That way the relationship between the label name and id is explicit, and if we ever use a different label it'll be a more clear and straightforward to update the code.

```
const schemaUpdateLabel = {
  name: 'schema update',
  id: 'MDU6TGFiZWwxMzQyMzM1MjQ2'
}
```

```suggestion
        pullRequests(first: 1, states: [OPEN], labels: [schemaUpdateLabel.name]) {
```

along with corresponding change below for the label id